src/tools/data/DataManualExamples/CMakeLists.txt

```diff
@@ -7,22 +7,18 @@
 #   Kathleen Biagas, Wed Jul 16 10:27:14 MST 2014
 #   Change how Simulations directory is handled on Windows.
 # 
+#   Kathleen Biagas, Thu Feb 27 2020
+#   Only add Simulations if VISIT_ENABLE_LIBSIM is true. 
+# 
 #****************************************************************************
 
-IF(NOT WIN32)
-    ADD_SUBDIRECTORY(CreatingCompatible)
-    ADD_SUBDIRECTORY(CreatingCompatibleAdvanced)
-    ADD_SUBDIRECTORY(Simulations)
-ELSE()
-    IF(VISIT_DATA_MANUAL_EXAMPLES)
-        ADD_SUBDIRECTORY(Simulations)
-    ELSE()
-        # The examples haven't been expressly requested, so exclude them
-        # from the VisIt solution.  A 'project' command in the Simulations
-        # CMakeLists.txt file will create a separate solution file and allow
-        # them to be built separately without requiring CMake to be re-run.
-        ADD_SUBDIRECTORY(Simulations EXCLUDE_FROM_ALL)
-    ENDIF()
-ENDIF(NOT WIN32)
+if(NOT WIN32)
+    add_subdirectory(CreatingCompatible)
+    add_subdirectory(CreatingCompatibleAdvanced)
+endif()
+
+if(VISIT_ENABLE_LIBSIM)
```

Comment:
Does `VISIT_ENABLE_LIBSIM` control *both* the plugin and the example sim?

Reply:
Yes, VISIT_ENABLE_LIBSIM will also control the plugin and the simulation examples due to their dependence on libsim.  Though the simulation examples will be turned off if VISIT_ENABLE_DATA_MANUAL_EXAMPLES is turned off, no matter if libsim is built or not.